docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -6,35 +6,59 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
 
 # Generating SessionSigs
 
-You can use any wallet or auth method to generate session signatures with the `getSessionSigs()` function from the Lit SDK. This function generates a session keypair and uses a callback function that signs the generated session key to create an `AuthSig` that is scoped to specific capabilities.
+The Lit SDK has three functions that explicitly generate session signatures: `getSessionSigs()`, `getPkpSessionSigs()`, and `getLitActionSessionSigs()`. These functions differ in their parameters and uses, but will all return session signatures. 
 
-In order to generate `SessionSigs`, you need a `capacityDelegationAuthSig` in signatures. This is because capacity credits are now required on both `Habanero` and `Manzano` networks. You can buy or [mint](../../capacity-credits.md#minting-capacity-credits) capacity credits centrally and [delegate](../../capacity-credits.md#delegating-access-to-your-capacity-credits-nft) them to your users. 
+These functions use the session keypair generated by the `LitNodeClient` to invoke a callback function that signs the generated session key, creating an `AuthSig` that is scoped to specific capabilities.
 
-You can generate a `SessionSig` with the help of `capacityDelegationAuthSig` object in the following way:
+In order to generate `SessionSigs` on both the `Habanero` and `Manzano` networks, you will need to include `capabilityAuthSigs`. This is because transactions on these networks require capacity credits. `capabilityAuthSigs` are **not** required on the `datil-dev` network.
```

Comment:
we might also want to reference the 'upcoming' datil-test and datil-prod networks as well here